plugins/Referrers/Widgets/GetCampaignUrlBuilder.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\Referrers\Widgets;
+
+use Piwik\Common;
+use Piwik\Piwik;
+use Piwik\Plugin;
+use Piwik\Widget\WidgetConfig;
+use Piwik\SettingsPiwik;
+
+class GetCampaignUrlBuilder extends \Piwik\Widget\Widget
+{
+    public static function configure(WidgetConfig $config)
+    {
+        $config->setCategoryId('Referrers_Referrers');
+        $config->setSubcategoryId('Referrers_URLCampaignBuilder');
+        $config->setName('Referrers_URLCampaignBuilder');
+        $config->setIsEnabled(SettingsPiwik::isSegmentationEnabled());
```

Comment:
Any reason for hiding the widget if segmentation is disabled?

Reply:
Nope, I'll update it.